docs/ReleaseProcess.md

```diff
@@ -25,41 +31,83 @@ eBPF for Windows, and how to service it later.
+1. Manually enter release notes or click "`Generate release notes`" and then edit as desired.
+1. Microsoft maintains an internal mirror of the eBPF for Windows project. For a given release branch in GitHub, the mirror repo will have a corresponding one.
+The release manager must download the "`ebpf-for-windows - MSI installer (Build-x64_Release)`" and "`ebpf-for-windows - NuGet package (Build-x64_Release)`" build
+artifacts from the Microsoft internal repository's build pipeline. Extract the `*.nupkg` file from it, and rename it to `eBPF-for-Windows.X.Y.0.nupkg`
+    - **NOTE** : The Microsoft internal build pipeline has two flavors of nuget package. The release manager must pick the one that *does not* contain "Redist" in
+    the name.
+1.  Attach the `Build-x64-[Release|Debug].zip`, the `Build-x64-native-only-[Release|Debug].X.Y.Z.zip`, the `.msi`, and the `.nupkg`, by dropping them in the
+"`Attach binaries by dropping them here or selecting them.`" area. For example, the file list for `v0.12.0` should be:
+    - *Build-x64-Debug.zip*
+    - *Build-x64-Release.zip*
+    - *Build-x64-native-only-Debug.0.12.0.zip*
+    - *Build-x64-native-only-Release.0.12.0.zip*
+    - *ebpf-for-windows.0.12.0.msi*
+    - *eBPF-for-Windows.0.12.0.nupkg*
 
-    - `eBPF-for-Windows.X.Y.Z.nupkg`
-1. Submit the PR for review (from its draft state), and wait for it to be approved and merged into the main repo's "`main`" branch.
-1. On the main `ebpf-for-windows` repo, create a new release branch from `main` **corresponding to the previous PR's commit**, name it "`release/X.Y`" (replace "X.Y" with the version number being released).
-1. Publish the release as per the "[Publishing a Release](ReleaseProcess.md#publishing-a-release)" process.
+1.  Check the "`Set as a pre-release`" checkbox, unless the release is production-signed.
+1.  Once the uploads are complete, click "`Publish release`".
 
-## Servicing a release
+## Publishing the Release to NuGet.org
+
+Upload the (**non-redist**) `.nupkg` file to [NuGet.org](https://www.nuget.org/) (the metadata inside the `.nuget` package itself will automatically populate all
+the other form fields).
+
+## Servicing a Release
 
 Servicing a release has two main scenarios:
 
 ### Updating a Release branch with patches/hot-fixes from main (*Forward Integration*)
 
 >NOTE: In servicing a release branch, **new features must not be added to the release branch**.  Only patches or hot-fixes will be accepted.
 
-1. On the main `ebpf-for-windows` repo, create and check out a new working branch from the `/release/X.Y` branch you want to service.
-1. Run the following script from the root directory of the repository, within a "*Developer Poweshell for VS 2022"* instance. Make sure to follow [Semantic Versioning 2.0](https://semver.org) ("`X.Y.Z`"):
+1. On the original `ebpf-for-windows` repo, create and check out a new topic branch from the `release/X.Y` branch you want to service.
```

Comment:
```suggestion
1. On the main `microsoft/ebpf-for-windows` repo, create and check out a new topic branch from the `release/X.Y` branch you want to service.
```

I would not use the word "original".

Reply:
that is the terminology used in (github doc)[https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo]. I could change it to `"upstream"`